.github/workflows/draft-release.yml

```diff
@@ -6,6 +6,8 @@ on:
 jobs:
   draft-release:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
I updated the repo "Actions permissions" settings to use the more restrictive "Read repository contents and package permissions" instead of the more permissive "Read and write permissions", thus this is now required.